packages/devops/ci/tupaia-ci-cd.Dockerfile

```diff
@@ -43,6 +51,8 @@ RUN mkdir -p ./packages/data-lake-api
 COPY packages/data-lake-api/package.json ./packages/data-lake-api
 RUN mkdir -p ./packages/database
 COPY packages/database/package.json ./packages/database
+RUN mkdir -p ./packages/devops
```

Comment:
I found that omitting some packages gives a different lockfile in Yarn2+, which means that the `yarn install --frozen-lockfile` below would fail, because it would want to change the lockfile. So I added them anyway. Adding these has no impact on CI/CD.